power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/common_solver_functions.hpp

```diff
@@ -8,7 +8,9 @@
 #include "y_bus.hpp"
 
 #include "../calculation_parameters.hpp"
+#include "../common/common.hpp"
 #include "../common/exception.hpp"
+#include <numeric>
```

Comment:
convention: space before 3rd party, system and STL libraries

```suggestion

#include <numeric>
```